website/docs/advanced-topics/server-side-rendering.md

```diff
@@ -39,8 +39,8 @@ fn App() -> Html {
     html! {<div>{"Hello, World!"}</div>}
 }
 
-#[tokio::main]
-async fn main() {
+// #[tokio::main]
+async fn no_main() {
```

Comment:
This works around the fact that there can't be two main functions, I assume?
Can you add a comment explaining this to the reader of the docs

Reply:
No, this is because we can't compile `#[tokio::main]` on WASM target (it defaults to `multi_thread`ed flavor). I fixed it by not enabling the `full` tokio feature and using the `current_thread` flavor